include/wx/webview.h

```diff
@@ -94,6 +94,15 @@ enum wxWebViewUserScriptInjectionTime
     wxWEBVIEW_INJECT_AT_DOCUMENT_END
 };
 
+enum wxWebViewBrowsingDataTypes
+{
+    wxWEBVIEW_BROWSING_DATA_COOKIES = 1,
+    wxWEBVIEW_BROWSING_DATA_CACHE = 2,
+    wxWEBVIEW_BROWSING_DATA_DOM_STORAGE = 4,
+    wxWEBVIEW_BROWSING_DATA_OTHER = 8,
+    wxWEBVIEW_BROWSING_DATA_ALL = 16
```

Comment:
I think this should actually be 15, i.e. the sum of everything below, as otherwise it's not really clear what is the difference between specifying all of the previous constants together or this one -- by making it equal to their sum, this would clearly show that this is equivalent.